### PR TITLE
Compensations for hardware testing

### PR DESCRIPTION
**Pull request by c-x-berger**
See also #12. This PR is for any changes made as a direct result of real-world testing of components.

This is being added to [DEEP SPACE](https://github.com/orgs/BHSSFRC/projects/2) and the stop build milestone as a tracker of when the changes are both done _and_ merged, because yay granularity!

### CODE REVIEW

**Comment by BoxWizard000000** [comment]
Looks good. Might be a bit jerky from what I see. I'll have to think of something to fix that if it is.

**Comment by c-x-berger** [comment]
It is a tad jerky (you can find video of it on Discord.) Best fix might be to just use a current ramp on the motor?

**Comment by c-x-berger** [comment]
> CAN ID of 0 now considered ‘unconfigured’ and will not be enabled.

:anger:

File: src/main/java/frc/robot/RobotMap.java [hunk 1]
```diff
@@ -25,7 +25,7 @@
     // public static final int rangefinderModule = 1;
 
     public class DRIVETRAIN {
-        public static final int LEFT_MASTER_CHANNEL = 0;
+        public static final int LEFT_MASTER_CHANNEL = 17;
```
Comment (c-x-berger) [inline src/main/java/frc/robot/RobotMap.java:28]: [Blame REV.](https://www.chiefdelphi.com/t/spark-max-firmware-release-v1-1-31/346537)

> CAN ID of 0 now considered ‘unconfigured’ and will not be enabled.

:anger:

**Comment by c-x-berger** [comment]
Leaving WIP status because we're about to wire the practice bot and make a whole new slew of these changes. (Merging once Travis agrees that things look good.)

**Pull request merged**